verifact/forum/migrations/0004_vote.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 3.1.6 on 2021-04-07 08:46
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
```

Comment:
Could merge 0004 and 0005 into a single migration to keep things clean, since they are very related.